jade/wsgi/routes/v1/judgements.py

```diff
@@ -2,49 +2,85 @@
 
 import flask
 
-from func_tools import wraps
+from ... import util
+from .... import errors, types
 
-from ... import responses
 
+def configure(config, bp, trusted_clients, centralauth, state):
 
-def configure(bp, trusted_clients, source):
+    @bp.route("/v1/judgements", methods=["GET"])
```

Comment:
There should be some general TODOs documented, e.g. we'll need result set paging.